### PR TITLE
IWF-397: Add channel sizes data

### PR DESCRIPTION
**Pull request by lwolczynski**

### CODE REVIEW

File: iwf/communication.py [hunk 1]
```diff
@@ -66,3 +79,40 @@ def get_to_publishing_internal_channel(self) -> list[InterStateChannelPublishing
...
+
+        if channel_name in self._to_publish_internal_channel:
+            buffer_channel_size = len(self._to_publish_internal_channel[channel_name])
+        else:
+            buffer_channel_size = 0
+
+        return server_channel_size + buffer_channel_size
+
+    def get_signal_channel_size(self, channel_name):
+        if (
```
Comment (longquanzheng) [inline iwf/communication.py:112]: I think we also need to check the signal type

**Review by longquanzheng — APPROVED** [review]
lgtm

File: iwf/communication.py (hunk 1 above)
Comment (lwolczynski) [reply iwf/communication.py:112]: Do you mean checking if the signal channel exists?

Communication object currently only stores internal channels in the `type_store`. I will need to extend it with `_signal_type_store` to do it

Comment (longquanzheng) [reply iwf/communication.py:112]: Yes. Oh that’s right because communication never needed to access that

**Review by samuel27m — APPROVED** [review]

**Pull request merged**